crates/router/src/core/admin.rs

```diff
@@ -660,10 +665,15 @@ pub async fn update_payment_connector(
 
     let frm_configs = match req.frm_configs.as_ref() {
```

Comment:
Move this logic to FRM crate and call it like 
```
let frm_configs = frm::utils::get_frm_configs(req.frm_configs);
```

Reply:
created function to help rewriting duplicate code 
```
pub fn get_frm_config_as_secret(
    frm_configs : Option<Vec<api_models::admin::FrmConfigs>>,
) -> Option<Vec<Secret<serde_json::Value>>> {
```